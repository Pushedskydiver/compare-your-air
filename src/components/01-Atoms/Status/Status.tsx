import React, { useContext } from 'react';
import StatusStyles from './Status.styles';
import Context from '../../Context';

const Status = () => {
  const { searchResults } = useContext(Context);
  const resultsLength = searchResults.length;

  return (
    <StatusStyles aria-live="polite" role="status">{resultsLength} results available.</StatusStyles>
  )
}

export default Status;
