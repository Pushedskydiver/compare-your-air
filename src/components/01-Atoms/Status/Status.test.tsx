import React from 'react';
import { cleanup, render } from '@testing-library/react';
import Status from './Status';

afterEach(() => {
  cleanup();
});

describe('<Status />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Status />
    );

    expect(container).toMatchSnapshot();
  });
})