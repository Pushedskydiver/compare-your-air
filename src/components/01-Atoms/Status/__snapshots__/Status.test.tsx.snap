// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Status /> should match snapshot 1`] = `
<div>
  <p
    aria-live="polite"
    class="sc-AxjAm fjFtQe"
    role="status"
  >
    0
     results available.
  </p>
</div>
`;
